Tests/Cache/NetworkingTests+Cache.swift

```diff
@@ -201,6 +208,15 @@ class CacheNetworkingTests: XCTestCase {
             }
         }
     }
+
+
+    private func systemPath(_ resource: String, extension ext: String) -> URL? {
```

Comment:
I think the `systemPath(_:extension:)` name is misleading since it might mean the path is for a resource in the user's machine, which is not necessarily true. Maybe `resourcePath(_:extension:)` would be better?